controllers/constants/image.go

```diff
@@ -1,33 +1,18 @@
 package constants
 
-import "runtime"
-
 const (
 	// OperandImageOpenJDK envvar name for OpenJDK operand image
 	OperandImageOpenJDK = "RELATED_IMAGE_OPENJDK"
-	// OperandImageOpenJ9 envvar name for OpenJ9 operand image
-	OperandImageOpenJ9 = "RELATED_IMAGE_OPENJ9"
 
 	// DefaultOperandImageOpenJDK default image for OpenJDK stack
 	DefaultOperandImageOpenJDK = "quay.io/infinispan/server:13.0"
-	// DefaultOperandImageOpenJ9 default image for OpenJ9 stack
-	DefaultOperandImageOpenJ9 = "infinispan-for-openj9/server:13.0"
 )
 
-// GetDefaultInfinispanJavaImage returns default Infinispan Java image depends of the runtime architecture
+// GetDefaultInfinispanJavaImage returns default Infinispan Java image
 func GetDefaultInfinispanJavaImage() string {
-	// Full list of archs might be found here:
-	// https://github.com/golang/go/blob/release-branch.go1.10/src/go/build/syslist.go#L8
-	switch arch := runtime.GOARCH; arch {
-	case "ppc64le", "s390x":
-		return GetEnvWithDefault(OperandImageOpenJ9, DefaultOperandImageOpenJ9)
-	default:
-		// Try new name RELATED_IMAGE_OPENJDK first
-		retVal := GetEnvWithDefault(OperandImageOpenJDK, "")
-		if retVal != "" {
-			return retVal
-		}
-		// If RELATED_IMAGE_OPENJDK is not defined fallback on old name for backward comp
-		return GetEnvWithDefault("DEFAULT_IMAGE", DefaultOperandImageOpenJDK)
+	if retVal := GetEnvWithDefault(OperandImageOpenJDK, ""); retVal != "" {
```

Comment:
As we're not using a default value here, we can just do:

```go
	if retVal, exists := os.LookupEnv(OperandImageOpenJ9); exists {
		return retVal
	}
	return GetEnvWithDefault("DEFAULT_IMAGE", DefaultOperandImageOpenJDK)
```

Reply:
Ignore that comment :slightly_smiling_face:. `os.LookupEnv` will return true if the env is set but empty, which is different to `GetEnvWithDefault`.

Reply:
ok, done